rdr_service/resource/generators/participant.py

```diff
@@ -1005,6 +1005,12 @@ def _make_sample_dict_from_row(bss=None, bos=None):
                     if bss.biobank_order_id == row.biobank_order_id:
                         bbo_samples.append(_make_sample_dict_from_row(bss=bss, bos=None))
                         stored_count += 1
+            # PDR-400: There are about 20 participants that have less ordered samples than stored samples.
+            elif len(bss_results) > 0 and len(bos_results) < len(bss_results):
```

Comment:
```suggestion
            elif (len(bss_results) > 0) and (len(bos_results) < len(bss_results)):
```

Reply:
Thanks Michael for the suggestion!  I think I'll pass on this suggestion though. :)

Reply:
😆 I just love commenting on RDR PR's. Trying to get that commit count back up.

Reply:
You could always come back...